<launch>
	<arg name="droneip" default="192.168.1.1"/>
	<node name="ardrone_driver" pkg="ardrone_autonomy" type="ardrone_driver" output="screen" args="-ip $(arg droneip)">
		<param name="navdata_demo" value="False"/>
		<param name="realtime_navdata" value="True"/>
		<param name="realtime_video" value="True"/>
		<param name="looprate" value="30"/>		
	</node>
	<node name="drone_gui" pkg="tum_ardrone" type="drone_gui"></node>
	
	<include file="$(find ar_track_alvar)/launch/Marktest_no_kinect.launch"/>
	
</launch>